drivers/wifi/nrf700x/osal/fw_if/umac_if/src/tx.c

```diff
@@ -584,6 +585,10 @@ int tx_cmd_prepare(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 	}
 
 	fmac_dev_ctx->host_stats.total_tx_pkts += config->num_tx_pkts;
+#ifdef DEBUG_MODE_SUPPORT
```

Comment:
Where does the `DEBUG_MODE_SUPPORT` define come from?

Have you considered using [`IF_ENABLED`](https://docs.zephyrproject.org/latest/kernel/util/index.html?highlight=if_enabled#c.IF_ENABLED) macro instead ifdefine?

Reply:
Removed DEBUG_MODE_SUPPORT from this PR since this was accidentally included with the Radio test code.